patterns.csv

```diff
@@ -1,8 +1,8 @@
 "Pattern"	"Label"	"Description"
-"[A-Z0-9]{2}-[A-Z0-9]{2}-[A-Z0-9]{2}"	"MAT"	"Matriculas portuguesas 0A-0A-0A"
+"(?<=[^\w-])(?:[A-Z]{2}-\d{2}-\d{2}|[A-Z]{2}-\d{2}-[A-Z]{2}|\d{2}-[A-Z]{2}-\d{2}|\d{2}-\d{2}-[A-Z]{2}|[A-Z]{2}\s?\d{2}\s?[A-Z]{2})(?![A-Z\d])(?=[^\w-])"	"MAT"	"Matriculas portuguesas 0A-0A-0A"
 "\d+(\-\w+)?\/(\w+\.)(\w+\.?)+"	"PRO"	"Processos com / (NN/...) opcionalmente apanha (NN-W/...) até ao próximo espaço"
-"\b\d{1,2}(-|\.|/)\d{1,2}(-|\.|/)\d{2,4}\b"	"DAT"	"Datas NN.NN.NNNN ou NN-NN-NNNN ou NN/NN/NNNN"
+"(?<=[^\w-])(?:[A-Z]{2}-\d{2}-\d{2}|[A-Z]{2}-\d{2}-[A-Z]{2}|\d{2}-[A-Z]{2}-\d{2}|\d{2}-\d{2}-[A-Z]{2}|[A-Z]{2}\s?\d{2}\s?[A-Z]{2}|\d{1,2}/\d{1,2}/\d{2,4}|\d{1,2}-\d{1,2}-\d{2,4})(?![A-Z\d])(?=[^\w-])"	"DAT"	"Datas NN.NN.NNNN ou NN-NN-NNNN ou NN/NN/NNNN"
```

Comment:
It looks like we are trying to solve everything on a single regex. I am not sure what this brings.

We can have multiple lines with different regexes pointing to the same entity type.

REG1 TYPE DESCRIPTION
REG2 TYPE DESCRIPTION

That might simplify the regex?